### ISSUE TITLE
Excuse me, how to count() Many-to-Many?

### ISSUE THREAD

**Issue by Wizzercn** [balderdashy/waterline#1189]
Sys_role.find()
            .where({id: roleid})
            .populate('users', {where: where, skip: start, limit: pageSize, sort: sort})
            .exec(function (err, obj) {
            });//paginate

How to count the number of users to meet the conditions? Without query()

**Comment by Wizzercn** [comment]
and ...How to query the user in addition to the above conditions?

.populate('users', {where: {'!',where}, skip: start, limit: pageSize, sort: sort}) ?..

**Comment by sailsbot** [comment]
Thanks for posting, @Wizzercn.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
